code/game/antagonist/antagonist.dm

```diff
@@ -8,6 +8,8 @@
 
 	var/datum/mind/owner = null
 
+	var/flags = 0                           // Various runtime options.
+
```

Comment:
Why that was added in prefs related PR?

Reply:
Ah yes, there is something you should know.
This flags used in relations to determinate is antag is valid candidate for them.
But these flags are not set anywhere but there is actually two places where same values for this flag is defined:
code\game\antagonist\_antagonist_setup.dm
and in 
code\__defines\gamemode.dm